lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -133,6 +133,7 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
   case Kinded::Kind::AvgPoolNodeKind:
   case Kinded::Kind::AdaptiveAvgPoolNodeKind:
   case Kinded::Kind::BatchedReduceAddNodeKind:
+  case Kinded::Kind::BatchedReduceSumSquareNodeKind:
```

Comment:
Is this necessary? The node is lowered and I don't think we need to report support for lowered nodes in `isOpSupported()`.